openhands/server/session/agent_session.py

```diff
@@ -75,10 +74,20 @@ async def start(
         self.thread = Thread(target=self._run, daemon=True)
         self.thread.start()
 
-        coro = self._start(runtime_name, config, agent, max_iterations, max_budget_per_task, agent_to_llm_config, agent_configs, status_message_callback)
-        asyncio.run_coroutine_threadsafe(coro, self.loop) # type: ignore
+        coro = self._start(
+            runtime_name,
```

Comment:
The changes to this file seem to be based on an older main branch, as these linter fixes were already applied on September 24.
I'd suggest to update your branch so there'd be no changes for this file in this PR needed as I don't see any Modal-related changes here?

Reply:
Yup done

Reply:
Thank you!